diff --git a/node_modules/wct-local/lib/browsers.js b/node_modules/wct-local/lib/browsers.js
index b9b4df8..0a86845 100644
--- a/node_modules/wct-local/lib/browsers.js
+++ b/node_modules/wct-local/lib/browsers.js
@@ -79,6 +79,7 @@ exports.detect = function detect(browserOptions) {
     return __awaiter(this, void 0, void 0, function* () {
         const launcher = yield util_1.promisify(launchpad.local)();
         const browsers = yield util_1.promisify(launcher.browsers)();
+        console.log(JSON.stringify(browsers));
         const results = {};
         for (const browser of browsers) {
             if (!LAUNCHPAD_TO_SELENIUM[browser.name]) {
